### PR TITLE
docs - update util ref pages for 9.0beta3

### PR DESCRIPTION
**Pull request by lisakowen**
in this PR:
- 9.0beta3 merge updates to utility commands.  most involve adding help- and version-related options.
- the existing pg_restore ref page does not document the "--use-set-session-authorization" option.   does pg_restore support this option?

### CODE REVIEW

File: gpdb-doc/dita/utility_guide/client_utilities/clusterdb.xml [hunk 1]
```diff
@@ -22,7 +22,15 @@ itself is extremely slow. </p><p>If you do need to order a table in this way to
           <codeph>clusterdb</codeph> would not be relevant.</p></section><section id="section4"><title>Options</title><parml><plentry><pt>-a | --all</pt><pd>Cluster all databases. </pd></plentry><plentry><pt>[-d | --dbname] <varname>dbname</varname> | [--dbname] <varname>dbname</varname></pt><pd>Specifies the name of the database to be clustered. If this is not
```
Comment (mkiyama) [inline gpdb-doc/dita/utility_guide/client_utilities/clusterdb.xml]: line 19-20 has "Greenplum recommends",  I think that should be changed to "Pivotal recommends" or be reworded to remove "recommends"

File: gpdb-doc/dita/utility_guide/client_utilities/createdb.xml [hunk 2]
```diff
@@ -32,8 +32,17 @@ containing white space must be enclosed in quotes.</pd></plentry><plentry><pt>-D
             use the default encoding. <ph>See the Greenplum Database Reference
```
Comment (mkiyama) [inline gpdb-doc/dita/utility_guide/client_utilities/createdb.xml]: Line 21 should lc-ctype be --lc-ctype (with --)?

File: gpdb-doc/dita/utility_guide/client_utilities/createdb.xml [hunk 3]
```diff
@@ -32,8 +32,17 @@ containing white space must be enclosed in quotes.</pd></plentry><plentry><pt>-D
...
+        </plentry>
+        <plentry>
+          <pt>-? | --help</pt>
+          <pd>Show help about <codeph>createdb</codeph> command line arguments, and exit.</pd>
+        </plentry>
+        </parml>
+      <p>The options <codeph>-D</codeph>, <codeph>-l</codeph>, <codeph>-E</codeph>, 
           <codeph>-O</codeph>, and <codeph>-T</codeph> correspond to options of the the underlying
         SQL command <codeph>CREATE DATABASE</codeph>; see <ph><xref
             href="../../ref_guide/sql_commands/CREATE_DATABASE.xml#topic1" otherprops="op-help"
```
Comment (mkiyama) [inline gpdb-doc/dita/utility_guide/client_utilities/createdb.xml]: line 46 has "the the"
line 48 xref to CREATE DATABASE should be in \<codeph>

File: gpdb-doc/dita/utility_guide/client_utilities/createdb.xml [hunk 4]
```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE topic
   PUBLIC "-//OASIS//DTD DITA Composite//EN" "ditabase.dtd">
-<topic id="topic1"><title id="oz137116">createdb</title><body><p id="client_util_desc">Creates a new database.</p><section id="section2"><title>Synopsis</title><codeblock id="client_util_synopsis"><b>createdb</b> [<varname>connection_option</varname> ...] [<b>-D</b> <varname>tablespace</varname>] [<b>-E</b> <varname>encoding</varname>] 
+<topic id="topic1"><title id="oz137116">createdb</title><body><p id="client_util_desc">Creates a new database.</p><section id="section2"><title>Synopsis</title><codeblock id="client_util_synopsis"><b>createdb</b> [<varname>connection-option</varname> ...] [<b>-D</b> <varname>tablespace</varname>] [<b>-E</b> <varname>encoding</varname>] 
         [<b>-O</b> <varname>owner</varname>] [<b>-T</b> <varname>template</varname>] [<b>-e</b>] [<varname>dbname</varname> ['<varname>description</varname>']]
 
-<b>createdb</b> <b>--help</b>
```
Comment (mkiyama) [inline gpdb-doc/dita/utility_guide/client_utilities/createdb.xml]: The options --locale, --lc-collate, --lc-type aren't in the syntax diagram.

File: gpdb-doc/dita/utility_guide/client_utilities/createlang.xml [hunk 5]
```diff
@@ -41,7 +41,7 @@
         </plentry>
```
Comment (mkiyama) [inline gpdb-doc/dita/utility_guide/client_utilities/createlang.xml]: line 31 has a link to postgres for PL/Java and PL/R, should that be pointed to the GPDB topics?

File: gpdb-doc/dita/utility_guide/client_utilities/vacuumdb.xml [hunk 6]
```diff
@@ -8,14 +8,15 @@
...
+    [<b>--analyze-only</b> | <b>-Z</b>]
 
-<b>vacuumdb</b> <b>--help</b> 
+<b>vacuumdb</b> <b>-? | --help</b>
 
-<b>vacuumdb</b> <b>--version</b></codeblock>
+<b>vacuumdb</b> <b>-V | --version</b></codeblock>
     </section>
     <section id="section3">
       <title>Description</title>
```
Comment (mkiyama) [inline gpdb-doc/dita/utility_guide/client_utilities/vacuumdb.xml]: In the description. 
"PostgreSQL" should be changed to "Greenplum"

line 23 vacuumdb should be in \<codeph.>

File: gpdb-doc/dita/utility_guide/client_utilities/psql.xml [hunk 7]
```diff
@@ -450,20 +457,25 @@ testdb=#</codeblock>
...
           <pd>This is not the actual command name: the letters <codeph>i</codeph>,
               <codeph>s</codeph>, <codeph>t</codeph>, <codeph>P</codeph>, <codeph>v</codeph>,
-              <codeph>E</codeph>, <codeph>S</codeph> stand for index, sequence, table, parent table,
-            view, external table, and system table, respectively. You can specify any or all of
-            these letters, in any order, to obtain a listing of all the matching objects. The letter
-              <codeph>S</codeph> restricts the listing to system objects; without
-            <codeph>S</codeph>, only non-system objects are shown. If + is appended to the command
-            name, each object is listed with its associated description, if any. If a pattern is
-            specified, only objects whose names match the pattern are listed.</pd>
+              <codeph>E</codeph>, <codeph>r</codeph> stand for index, sequence, table, parent table,
```
[the remark this answers is not in the document]
Comment (lisakowen) [reply gpdb-doc/dita/utility_guide/client_utilities/psql.xml]: good catch.  this didn't work for me either.  the postgres 9.4 docs indicate that \dE should display foreign tables.  \dE in greenplum displays external tables, of which foreign tables are one kind.  i will replace the \dr references with \det, likely in a new option/param block.

**Pull request merged**